### PR TITLE
Updated SQL for whole database dump based on LSE feedback

### PR DESCRIPTION
**Pull request by katybaulch**
# Description

Updated SQL for whole database dump based on LSE feedback.
See [this PR](https://github.com/climatepolicyradar/navigator-backend/pull/208) for more info.

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## Reviewer Checklist

- [x] The PR represents a single feature (small driveby fixes are also ok)
- [x] The PR includes tests that are sufficient for the level of risk
- [x] The code is sufficiently commented, particularly in hard-to-understand areas
- [x] Any required documentation updates have been made
- [x] Any TODOs added are captured in future tickets
- [x] No FIXMEs remain

### CODE REVIEW

File: app/core/download.py [hunk 1]
```diff
@@ -175,7 +172,10 @@ def create_query() -> str:
     n3.event_type_names as "Full timeline of events (types)",
     n3.event_dates as "Full timeline of events (dates)",
     d.created::date as "Date Added to System",
-    f.last_modified::date as " Last Modified on System"
+    f.last_modified::date as " Last Modified on System",
```
Comment (olaughter) [inline app/core/download.py:175]: Theres a bit of extra whitespace here 🔎

File: app/core/download.py [hunk 2]
```diff
@@ -270,7 +270,6 @@ def convert_dump_to_csv(df: pd.DataFrame):
     return csv_buffer
 
 
-def generate_data_dump_as_csv(app_url: str, db=Depends(get_db)):
+def generate_data_dump_as_csv(db=Depends(get_db)):
     df = get_whole_database_dump(db)
-    df = replace_slug_with_qualified_url(df, app_url)
```
Comment (olaughter) [inline app/core/download.py:275]: Does `replace_slug_with_qualified_url` no longer get called? in which case it can be deleted

**Review by olaughter — APPROVED** [review]
lgtm. A couple of teeny points below

File: app/core/download.py (hunk 2 above)
Comment (katybaulch) [reply app/core/download.py:275]: We'll need it again soon! For the first draft of this data dump work it's been decided not to include fully qualified URLs, but I think we are wanting them at some point. I'll pop a comment on the function.

File: app/core/download.py (hunk 1 above)
Comment (katybaulch) [reply app/core/download.py:175]: No whitespace here - the line has been moved 😄

Comment (katybaulch) [reply app/core/download.py]: Whoops - see what you mean. Have fixed.

**Review by diversemix — APPROVED** [review]
Good iterative approach :100:

**Pull request merged**